featureFlags/README.md

```diff
@@ -0,0 +1,149 @@
+# FeatureFlags
+
+# FeatureFlag object
+
+```
+{
```

Comment:
Please check the update to the schema of this object in the linked issue for this task